### ISSUE TITLE
window.crypto.randomUUID is not a function

### ISSUE THREAD

**Issue by dvargasfuertes** [paulovieira/roam-reference-path#14]
Hey,

This extension has a Mutation Observer that is crashing on the SmartBlocks settings page, preventing users from using SmartBlocks, with the error message mentioned in the title. Any chance we could prioritize this issue?

Steps to reproduce:
1. Install Reference Path
2. Install SmartBlocks
3. Toggle on daily from the smartblocks extension

**Comment by paulovieira** [comment]
Sorry about this. It's late here but I'll see this tomorrow morning. I can easily replace that uuid generator with something else. I thought it was a standard thing in browsers by now.

**Comment by paulovieira** [comment]
I already had a PR in roam depot waiting to be approved. I'll update the commit hash there in a few minutes.

**Comment by paulovieira** [comment]
@dvargas92495 By the way, the new version is not yet committed to roam depot, but I was able to have both extensions working without any error. 

I reproduced the steps in your initial message and got the SmartBLock workflow in the daily page, as expected. No errors in the devtools console.

Even so, someone might be having problems with the `crypto.randomUUID` method so it's a good idea to replace it. But I think the problem might be in fact that the "daily" feature is only executed once per day (right?). My guess is that someone expected the daily workflow to run again. It didn't happen and the error related to `randomUUID` was seen in the console, and the connection was made.

I don't see how some error in this extension (or any other) could stop SmartBlocks from doing its thing.
